doc/classes/HTTPRequest.xml

```diff
@@ -309,6 +309,7 @@
 			Request reached its maximum redirect limit, see [member max_redirects].
 		</constant>
 		<constant name="RESULT_TIMEOUT" value="13" enum="Result">
+			Request failed due to a timeout. Request took longer to complete than the value declared at [member timeout].
```

Comment:
```suggestion
			Request failed due to a timeout. If you expect requests to take a long time, try increasing the value of [member timeout] or setting it to [code]0.0[/code] to remove the timeout completely.
```

The second sentence is a bit awkward. It steps on the heels of the previous one ("Request failed due to a timeout. Request took longer ..."), and the ending also doesn't sound correct ("the value declared at"). I think changing it to this suggestion would work better.